debian/libmavsdk0.install

```diff
@@ -0,0 +1 @@
+usr/lib/*/lib*.so.*
```

Comment:
What's the meaning of the 0 in the filename `libmavsdk0.install` here?

Reply:
this is the so major version -- distinction at package name level allows to co-install incompatible major versions of the runtime library, i.e. when you go to 1.0 package name should change to libmavsdk1

Reply:
Got it. I see how this can make sense, however, currently for all other deployments, it's just mavsdk, e.g. on homebrew, AUR, and for the .deb and .rpm packages.

Reply:
so dropping the 0 from name? sure can do that, ultimately it depends on how you want to deal with compatibility

Reply:
I tend to vote to drop the 0, mainly for consistency with other installations. But then at linking time it should still fail given the soname. @JonasVautherin what do you think?

Reply:
not sure you are going to get too much consistency with any other installation -- right now structure is fully consistent with lib packaging guidelines, i.e. the following set of packages are produced:

libmavsdk0 -- runtime shared libraries
libmavsdk0-dbgsym -- debug symbols for runtime libraries
libmavsdk-dev -- development header files (and maybe if you like at some point, static libs)

there would be additional packages generated for mavlink ftp client + server, but not done yet

dropping version number is however not an issue, and probably makes sense as long as nobody cares too much about binary compatibility

Reply:
I don't have a very strong opinion, but I would tend towards what's standard on the platform. My intuition would be that if I'm on my Ubuntu, I like having consistency between the packages on my Ubuntu, and I don't mind so much if macOS or Windows work differently. So if it's normal on Ubuntu to call it libmavsdk0, I would maybe go for that. But again, not feeling strongly about it :slightly_smiling_face:.

Reply:
Ok, that's fine with me. So question back to @caleridas what's common or recommended on Debian/Ubuntu?

Reply:
common use is libfoo$VERSION or libfoo-$VERSION -- where VERSION is either major version, or may be down to minor version -- it is however very rare that anyone needs minor versions co-installed

some older packaged libraries lack the $VERSION indicator and are fine with it as they sometimes have not had incompatible changes in 20+ years

Reply:
Ok, we'll go with your suggetion then, thanks @caleridas!